src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -653,6 +656,14 @@ public static void updateConfigErrorMessage(Config result, String key, String ms
    * @return log message wrapped by snowflake tag
    */
   public static String formatLogMessage(String format, Object... vars) {
+    // note this MDC context is only available for apache kafka versions after 2.3.0, more
+    // information here:
+    // https://cwiki.apache.org/confluence/display/KAFKA/KIP-449%3A+Add+connector+contexts+to+Connect+worker+logs
+    String connCtx = MDC.get(MDC_CONN_CTX_KEY);
+    if (connCtx != null && !connCtx.isEmpty()) {
+      return SF_LOG_TAG + " " + connCtx + formatString(format, vars);
```

Comment:
now that I think more about this, I do feel about adding this connector context behind a param, with default being set.
Couple of reasons: 
1. They can turn it off at SF KC level. 
2. If it's not set at Kafka Connector level, our default true will help them. 

They might not like the default true option to begin with but if they are super worried about redundant logs (from KC and SF KC connector context) they will switch it off. Turning off at KC level would not be a great option for them since it will affect other connectors. 

WDYT?

Reply:
added param that defaults to false. thanks for the suggestion!